process_functions.py

```
@@ -1,3 +1,910 @@
+                    None
+    print("Indexing thread going down!")
+    global_settings.processados.put(('clear_searches', True))
+
+
+def run_image_image_match(diretorio_temp_input, diretorio_temp_output, abs_path_pdf, idpdf):
+    image_match_dir = os.path.join(utilities_general.get_application_path(), 'image_match')
+    image_match_exe_abs = os.path.join(image_match_dir, 'image_match.exe')
+   # cmd = r"{} calc_descriptor33 {} {}".format(image_match_exe_abs, diretorio_temp_input, diretorio_temp_output)
+    cmd = "\"{}\" calc_descriptor33 \"{}\" \"{}\"".format(image_match_exe_abs, diretorio_temp_input, diretorio_temp_output)
+    #print(cmd)
+    popen = subprocess.Popen(cmd, universal_newlines=True, shell=True, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
```

Ruff (0.15.10)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)